phpBB/includes/functions.php

```diff
@@ -4181,7 +4181,7 @@ function page_header($page_title = '', $display_online_list = false, $item_id =
 	}
 	else
 	{
-		$u_login_logout = append_sid("{$phpbb_root_path}ucp.$phpEx", 'mode=login');
+		$u_login_logout = append_sid("{$phpbb_root_path}ucp.$phpEx", 'mode=login&amp;redirect=' . rawurlencode($user->page['page']));
```

Comment:
What happens if I enter my password wrong and then enter it correctly?

Reply:
Done (i think)

Reply:
landed on https://github.com/phpbb/phpbb when I was logged into Github

Reply:
Again nothing to do with this pull request. This pr is for the phpBB software and wouldn't redirect you to GitHub.

Reply:
Yeah, I figured that out when I saw where the page landed..
I really like the emotion buttons. That's why I want to learn more about this forum.

Reply:
```php
// if user enter password wrong and then correct
$redirect = !empty($request->variable('redirect', '')) ? $request->variable('redirect', '') : $user->page['page'];
$u_login_logout = append_sid("{$phpbb_root_path}ucp.$phpEx", 'mode=login&amp;redirect=' . rawurlencode($redirect));